tests/integration/test_parallel_inception.py

```diff
@@ -0,0 +1,79 @@
+from __future__ import print_function, unicode_literals
+
+import subprocess
```

Comment:
I appreciate you've tried to do everything from scratch, but can you instead use our battle tested pytest fixture; that significantly simplifies the amount of code - see https://github.com/tox-dev/tox/blob/master/tests/unit/session/test_parallel.py#L15-L36 e.g.

Reply:
I've only seen initproj and it seemed like it does files I don't need. Will try with cmd

Reply:
amended